src/main/java/core/basesyntax/db/Storage.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.db;
+
+import core.basesyntax.model.Fruit;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<Fruit, Fruit> storage = new HashMap<>();
```

Comment:
Why are you using two identical objects in a key/value pair?

Reply:
I tried to simulate the behaviour of DB as it stores entities.

Reply:
let's better use `Set<Fruit>`
Map doesn't look good

Reply:
Corrected